src/hooks/useIncludes.test.js

```diff
@@ -18,32 +17,32 @@ describe('useIncludes hook', () => {
 
     expect(() => {
       isLegalSizeValue('nope')
-    }).toThrow(/Invalid item \'nope\'/)
+    }).toThrow(/Invalid item 'nope'/)
   })
 
-  test.only('valid single keys do not throw', () => {
+  test('valid single keys do not throw', () => {
     const whitelist = ['yogabba']
     const [, includesKeysOrThrow] = useIncludes(whitelist)
     expect(includesKeysOrThrow({ yogabba: 'y' })).toBeUndefined()
   })
 
-  test.only('invalid single key throws', () => {
+  test('invalid single key throws', () => {
     const whitelist = ['yogabba']
     const [, includesKeysOrThrow] = useIncludes(whitelist)
     expect(() => {
       includesKeysOrThrow({ nope: 'n' })
     }).toThrow(/Unexpected props: 'nope'/)
   })
 
-  test.only('validates multiple keys', () => {
+  test('validates multiple keys', () => {
     const whitelist = ['a', 'b', 'c']
     const [, includesKeysOrThrow] = useIncludes(whitelist)
     expect(includesKeysOrThrow({ a: 'valid' })).toBeUndefined()
     expect(includesKeysOrThrow({ b: 'valid' })).toBeUndefined()
     expect(includesKeysOrThrow({ c: 'valid' })).toBeUndefined()
   })
 
-  test.only('validates multiple keys and throws if invalid', () => {
```

Comment:
This OK @roblevintennis? Not sure if you wanted all these tests running. They pass.

Reply:
No it's a blunder...thanks. I can't wait to start religiously running `yarn lint` lol